api/nakadi-event-bus-api.yaml

```diff
@@ -1203,10 +1204,10 @@ definitions:
         items:
           type: string
         description: |
-          Indicator of the field used for guaranteeing the ordering of Events of this type (used only by 'hash'
+          Indicator of the fields used for guaranteeing the ordering of Events of this type (used only by 'hash'
           PartitionResolutionStrategy).
           If set MUST be a valid required field as defined in the schema.
-          It MUST be set if 'partition_resolution_strategy' is set to 'hash'.
+          It MUST be set if 'partition_resolution_strategy' is set to 'hash'. In other cases it is ignored.
```

Comment:
You could also use the same wording as before "In case of another partitioning strategy it is ignored.". Its very clear that way.